lib/data-service.js

```diff
@@ -286,6 +286,37 @@ class DataService extends EventEmitter {
   }
 
   /**
+   * Returns the results of currentOp.
+   *
+   * @param {Object} operations - Document containing the operations to return information about.
+   * @param {Function} callback - The callback.
+   */
+  currentOp(operations, callback) {
+    debug(`#currentOp: ${operations}`);
+    this.client.currentOp(operations, callback);
```

Comment:
Here you are passing an operations object but the `NativeClient` is expecting a `Boolean`. I would change `operations` to `includeAll` as you have in the native client.